ui/tract-renderer/src/main/java/org/cru/godtools/tract/analytics/model/ToggleLanguageAnalyticsActionEvent.kt

```diff
@@ -1,25 +1,22 @@
 package org.cru.godtools.tract.analytics.model
 
+import android.os.Bundle
 import java.util.Locale
 import org.ccci.gto.android.common.compat.util.LocaleCompat.toLanguageTag
-import org.cru.godtools.analytics.adobe.ADOBE_ATTR_LANGUAGE_SECONDARY
+import org.cru.godtools.analytics.firebase.PARAM_LANGUAGE_SECONDARY
 import org.cru.godtools.analytics.model.AnalyticsActionEvent
 import org.cru.godtools.analytics.model.AnalyticsSystem
 
-private const val ACTION_TOGGLE_LANGUAGE = "Parallel Language Toggled"
-
-private const val ADOBE_ATTR_TOGGLE_LANGUAGE = "cru.parallellanguagetoggle"
+private const val ACTION_TOGGLE_LANGUAGE = "parallel_language_toggled"
 
 class ToggleLanguageAnalyticsActionEvent(tool: String?, locale: Locale) :
     AnalyticsActionEvent(action = ACTION_TOGGLE_LANGUAGE) {
     override fun isForSystem(system: AnalyticsSystem) =
-        system === AnalyticsSystem.ADOBE || system === AnalyticsSystem.FACEBOOK
+        system === AnalyticsSystem.FIREBASE || system === AnalyticsSystem.FACEBOOK
 
     override val appSection = tool
     @OptIn(ExperimentalStdlibApi::class)
```

Comment:
you can actually remove this `@OptIn` annotation, it was present for the use of `buildMap` which is no longer being used.